schema.sql

```diff
@@ -1,10 +1,60 @@
 /* Database schema to keep the structure of entire database. */
 
+DROP TABLE IF EXISTS animals;
+
 CREATE TABLE animals (
     id int PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
     name char(50),
     date_of_birth date,
     escape_attempts int,
     neutered boolean,
-    weight_kg decimal
+    weight_kg decimal,
+    species char(50)
```

Comment:
* Kindly add species column the right way [Read up on how to properly add a column](https://www.w3schools.com/sql/sql_alter.asp)